File: ios/Records/Delays.swift
import ExpoModulesCore

struct Delays: Record, Equatable {
    @Field
    var audio: Int = 0

    @Field
    var subtitle: Int = 0

    // swiftformat:disable:next redundantEquatable
    static func == (lhs: Delays, rhs: Delays) -> Bool {
        lhs.audio == rhs.audio &&
            lhs.subtitle == rhs.subtitle
    }
}

File: android/src/main/java/expo/modules/libvlcplayer/records/Delays.kt
package expo.modules.libvlcplayer.records

import expo.modules.kotlin.records.Field
import expo.modules.kotlin.records.Record
import java.io.Serializable

class Delays(
  @Field var audio: Long = 0,
  @Field var subtitle: Long = 0,
) : Record,
  Serializable
